docs/_articles/en/v3.x/salesforce-graph-engine/try-it-yourself.md

```diff
@@ -18,7 +18,7 @@ Next, run a command to view Graph Engine rules, then install our sample project
 To see Graph Engine’s rules, run:
 
 ```bash
-sfdx scanner:rule:list --engine sfge
+sf scanner rule list --engine sfge
 ```
 
 Currently, just one rule is returned. [ApexFlsViolationRule](./en/v3.x/salesforce-graph-engine/rules/#apexflsviolationrule) identifies Create Read Update Delete/Field-Level Security (CRUD/FLS) vulnerabilities in your Apex code.
```

Comment:
This line is no longer correct, since we have a bunch of GraphEngine rules. Can we (not necessarily with this PR) update it to be more future-proof, e.g., "There are a number of rules, but this document will be focusing on ApexFlsViolationRule"?